### PR TITLE
switch out connect http module for express.

### PR DESCRIPTION
**Pull request by nsaenz7**
Upgrade our http module. Remove `connect` references in the codebase / comments to minimize confusion.

### CODE REVIEW

**Review by srksag-zz — DISMISSED** [review]

File: lib/startUpServer.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ function startUpServer(preSessionInject, postSessionInject, appList, appStartLis
     let appContext = AmorphicContext.appContext;
     let appConfig = AmorphicContext.applicationConfig[mainApp];
     let reqBodySizeLimit = appConfig.reqBodySizeLimit || '50mb';
-    let app = connect();
+    let app = express();
```
Comment (hackerrdave) [inline lib/startUpServer.js:44]: on line 116 - is `appContext.connectServer` a reference to Connect? if so should we update that to `appContext.expressServer` ? or just simply `appContext.server`?

File: lib/getController.js [hunk 2]
```diff
@@ -17,23 +17,23 @@ let decompressSessionData = require('./session/decompressSessionData').decompres
...
- * @param {unknown} connectSession - connect session object
+ * @param {unknown} expressSession - express session object
  * @param {unknown} objectCacheExpiration - seconds to expire controller object cache
  * @param {unknown} sessionStore - session implementation
  * @param {Boolean|String} newPage - force returning everything since this is likely a session continuation on a
  * new web page.
  * @param {unknown} reset - create new clean empty controller losing all data
  * @param {unknown} controllerId - unknown
- * @param {Object} req - Connect request object.
+ * @param {Object} req - express request object.
```
Comment (hackerrdave) [inline lib/getController.js]: nit: maybe use capital "E" to refer to Express for clarity?

File: lib/startUpServer.js (hunk 1 above)
Comment (nsaenz7) [reply lib/startUpServer.js:44]: seems to be - updated to just be `appContext.server`.

**Review by hackerrdave — DISMISSED** [review]

**Review by nikmash — APPROVED** [review]
Looks good!

**Review by sshankar19 — APPROVED** [review]

**Review by hackerrdave — APPROVED** [review]

**Review by sshankar19 — APPROVED** [review]

**Review by hackerrdave — APPROVED** [review]

**Pull request merged**